### PR TITLE
v1/customer: Add another API

### PR DESCRIPTION
**Pull request by nkristianto**
Add another customer API definition. Fix tag on customer.

### CODE REVIEW

File: v1/customer/customer.gunk [hunk 1]
```diff
@@ -341,6 +341,124 @@ type GetTaxResidenceOfCustomerResponse struct {
...
+type UpdateCustomerEmailResponse struct {
+	Customer Customer `pb:"1" json:"customer"`
+}
+
+type UpdateCustomerIdentityRequest struct {
+	BankID      string `pb:"1" json:"bank_id"`
+	CustomerID  string `pb:"2" json:"customer_id"`
+	LegalName   string `pb:"3" json:"legal_name"`
+	DateOfBirth string `pb:"4" json:"date_of_birth"`
+	Title       string `pb:"5" json:""`
```
Comment (ghazninattarshah) [inline v1/customer/customer.gunk]: json tag is empty

File: v1/customer/customer.gunk [hunk 2]
```diff
@@ -893,4 +1011,346 @@ type CustomerService interface {
...
+	//                                 "OAuth2": openapiv2.SecurityRequirement_SecurityRequirementValue{
+	//                                         Scope: []string{
+	//                                                 "write",
+	//                                         },
+	//                                 },
+	//                         },
+	//                 },
+	//         },
+	// }
+	UpdateCustomerAddress(UpdateCustomerAddressRequest) UpdateCustomerAddressResponse
```
Comment (ghazninattarshah) [inline v1/customer/customer.gunk]: Ref: customerUpdateCustomerAddressResult
Actual Return: UpdateCustomerAddressResponse

Update the reference with actual return type

**Review by ghazninattarshah — CHANGES_REQUESTED** [review]
Looks like the openapi/response/schema/tag/ref does not match with the actual response of the service function. @nkristianto

File: v1/customer/customer.gunk (hunk 2 above)
Comment (nkristianto) [reply v1/customer/customer.gunk]: whooppss, thank you

**Review by ghazninattarshah — APPROVED** [review]
LGTM

**Review by jwangsadinata — APPROVED** [review]
LGTM, thanks @nkristianto.

**Pull request merged**